core-plugin/src/com/google/gct/idea/debugger/ProjectRepositoryValidator.java

```diff
@@ -123,19 +145,36 @@ public SyncResult checkSyncStashState() {
     }
 
     if (!foundDebuggee) {
-      return new SyncResult(/*isinvalid*/true,
+      return new SyncResult(/*isinvalid*/ true,
                             /*needsstash*/ false,
                             /*needssync*/ false,
-                            /*isdeterminable*/ true,
                             /*target SHA*/ null,
-                            /*target repo*/ null);
+                            /*target repo*/ null,
+                            /* hasCloudRepository */ false,
+                            /* repoType */ "Unknown");
```

Comment:
On Mon, Nov 16, 2015 at 9:49 AM, Patrick Flynn <notifications@github.com>
wrote:

> do you mean along a url path? Otherwise anything that's displayed as
> language to the user should be read from resource bundles.


done


-- 
Elliotte Rusty Harold
elharo@macfaq.com